app/helpers/client_data_helper.rb

```diff
@@ -0,0 +1,26 @@
+module ClientDataHelper
```

Comment:
`ClientHelper` actually seems more appropriate here, since these methods really have little to do with the proposal extension values themselves.